src/table/utils/selections-utils.js

```diff
@@ -110,7 +115,7 @@ const selectCell = (state, payload) => {
   handleAnnounceSelectionStatus({
     announce,
     rowsLength: selectedRowsLength,
-    isAddition: selectedRowsLength > rows.length,
+    isAddition: selectedRowsLength >= Object.keys(rows).length,
```

Comment:
hmm is there a case where you shift+arrow to an an already selected value, so the number of selected values does not change? That would probably be a special case for the announcer.

Reply:
Correct. I added shouldAnnounce here
 https://github.com/qlik-oss/sn-table/pull/555/commits/c4b905776acba6a20166f958d138b140e5c83c38#diff-3f227d6a51355e588ecb2a10cd87fc83d3c270d50b08fe3cb560baa979976aeaR86